source/packages/services/commands/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "clean": "npx shx rm -rf dist tsconfig.tsbuildinfo bundle.zip .rush .nyc_output *.log",
     "lint": "npx eslint . --ext '.ts'",
-    "build": "npx tsc -b && npx shx cp -R 'src/config' 'dist/'",
+    "build": "npx tsc -p tsconfig.json && npx shx cp -R 'src/config' 'dist/'",
```

Comment:
Can you add a note about why this change exists?

Reply:
This is an attempt to make `command` package pass the build. The rest of the packages use `npx tsc -p tsconfig.json` for build, so I kept this change in the PR.